docs/ref/databases.txt

```diff
@@ -941,6 +941,27 @@ To enable the JSON1 extension you can follow the instruction on
 .. _JSON1 extension: https://www.sqlite.org/json1.html
 .. _the wiki page: https://code.djangoproject.com/wiki/JSON1Extension
 
+.. _sqlite-init-command:
+
+Setting PRAGMA Options
+----------------------
+
+.. versionadded:: 5.1
+
+Pragma options can be set upon connection by using the ``init_command`` option
+similar to the MySQL backend. The example below shows how to disable
+synchronous writes::
+
+    DATABASES = {
+        "default": {
+            "ENGINE": "django.db.backends.sqlite3",
+            "NAME": BASE_DIR / "db.sqlite3",
+            "OPTIONS": {
+                "init_command": "PRAGMA synchronous=0;",
```

Comment:
Can we change the example to `journal_mode=WAL` instead of `synchronous=0`? `synchronous=0` could corrupt the database on system crash/power loss, so mindlessly copy-pasting the example could lead to data loss.

I'd prefer to see `journal_mode=WAL` here since I think it will be a common thing to set, but `synchronous=3` would also be okay since it increases durability.

```suggestion
Pragma options can be set upon connection by using the ``init_command`` option
similar to the MySQL backend. The example below shows how to enable WAL
journaling mode::

    DATABASES = {
        "default": {
            "ENGINE": "django.db.backends.sqlite3",
            "NAME": BASE_DIR / "db.sqlite3",
            "OPTIONS": {
                "init_command": "PRAGMA journal_mode=WAL;",
```

Reply:
There was some [feedback in trac](https://code.djangoproject.com/ticket/24018#comment:8) that noted that since `journal_mode=WAL` is persistent, that setting it once via the CLI would be sufficient.

I don't have a strong preference (the original PR used WAL in the example) and will defer to others on this one (and agree on reasons for showing sync=3 instead of sync=0 if that option is preferred too).

Reply:
Yes, please change to the `synchronous=3`.

Reply:
Done